CHANGELOG.rst

```diff
@@ -1,3 +1,10 @@
+3.13.0
+======
+
+Bug Fixes
+---------
+* Support retry_policy in PreparedStatement
```

Comment:
Can you add here the number of the ticket, like
Support retry_policy in PreparedStatement (PYTHON-861)

Reply:
done